MaxText/layers.py

```diff
@@ -39,8 +39,14 @@
 from jax.ad_checkpoint import checkpoint_name
 import jax.numpy as jnp
 from jax.experimental.pallas.ops.tpu import flash_attention
+import os
+import math
 
-
+try:
+  from jax.experimental.pallas.ops import attention
+  from jax.experimental.pallas.ops import layer_norm
+except ImportError:
+  logging.warning('jax_triton not found, please `pip install jax-triton`')
```

Comment:
I'd really prefer to not crash for all users here! Can we crash when we use these things if they're not installed.